king_phisher/client/windows/plugin_manager.py

```diff
@@ -339,13 +350,39 @@ def _plugin_enable(self, model_row):
 		self._set_model_item(model_row.path, enabled=True, sensitive_installed=False)
 		self.config['plugins.enabled'].append(named_row.id)
 
+	def _compatibility_to_dict(self, compatibility_details):
```

Comment:
I don't even see where this function is referenced, if it's not used please remove the whole thing.